packages/react-components/react-badge/src/components/PresenceBadge/usePresenceBadgeStyles.ts

```diff
@@ -1,4 +1,4 @@
-import { mergeClasses, makeStyles, shorthands } from '@griffel/react';
+import { mergeClasses, makeResetStyles, makeStyles } from '@griffel/react';
```

Comment:
_nit:_ Take this opportunity to order these alphabetically

```suggestion
import { makeResetStyles, makeStyles, mergeClasses } from '@griffel/react';
```